app/controllers/api/greetings_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::GreetingsController < ApplicationController
+  def random_greeting
+    @greeting = Greeting.order('RANDOM()').first
+
+    render json: @greeting
+  end
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you render only the `message` attribute instead of the entire record

   ![hold](https://user-images.githubusercontent.com/36542986/184873643-163061c3-19c7-4928-8abf-993760ede4b8.jpg)